### PR TITLE
add myself as code owner for xclbinutil

### PR DESCRIPTION
**Pull request by xfreid**
#### Problem solved by the commit
Add myself as code owner for xclbinutil

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
None

#### How problem was solved, alternative solutions (if any) and why they were rejected
N/A

#### Risks (if any) associated the changes in the commit
None

#### What has been tested and how, request additional testing if necessary
None

#### Documentation impact (if any)
None

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -37,6 +37,7 @@ src/runtime_src/xocl/api/plugin/xdp/ @jvillarre
 src/runtime_src/xdp/ @jvillarre
 src/runtime_src/doc/ @uday610
 src/runtime_src/tools/ @rozumx
+src/runtime_src/tools/xclbinutil/ @feirui
```
Comment (chvamshi-xilinx) [inline .github/CODEOWNERS:40]: Hi Fei , i see your username is @xfreid .  you added @feirui here. i am not sure whether this works or not.

Comment (stsoe) [reply .github/CODEOWNERS:40]: This will not work.  See the underlined error.

**Review by stsoe — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (xfreid) [reply .github/CODEOWNERS]: My bad, just fixed it.

**Review by stsoe — CHANGES_REQUESTED** [review]
When you specialize a subdirectory you change the rule to what you write for that subdir.  E.g. previously src/runtime_src/tools/xclbinutil/ was covered by src/runtime_src/tools/  which had @rozumx as a reviewer.  Now with your addition xclbintuil/ is speciailized and only notifies you.  You probably should add @rozumx as well.  See how tools/scripts/ is specialized .

**Review by stsoe — APPROVED** [review]

**Pull request merged**